permissions/plugin-azure-vmss.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "azure-vmss"
+paths:
+- "org/jenkins-ci/plugins/azure-vmss"
+developers:
+- "chenkennt"
+- "azure_devops"
```

Comment:
`azure_devops` has never logged in to Artifactory. This blocks applying these permissions, see this repo's readme for details.

Reply:
Just logged in.